examples/getStealthMetaAddress/index.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  createStealthClient,
+  getStealthMetaAddress,
+  VALID_SCHEME_ID,
+  ERC6538_CONTRACT,
+} from 'stealth-address-sdk';
+
+// Example stealth client parameters
+const chainId = 11155111; // Example chain ID for Sepolia
+const rpcUrl = 'https://sepolia.infura.io/v3/YOUR_INFURA_PROJECT_ID';
```

Comment:
This key should be taken out into a `.env.exmaple`

Reply:
included [here](https://github.com/ScopeLift/stealth-address-sdk/pull/19/commits/00391af1679c789884a727a5dbb6b8274489c9cf)